### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by haydnba**
This add very basic signup functionality - no validation etc.

### CODE REVIEW

**Review by n1c0z — APPROVED** [review]
:+1:  :smile:  :100:

**Review by iPhatty — APPROVED** [review]
👍

File: src/model/database/db_build.js [hunk 1]
```diff
@@ -10,6 +10,6 @@ const build = sql('./db_build.sql');
 // database building for test
 const runDbBuild = () => db.query(build);
 
-runDbBuild();
+// runDbBuild();
```
Comment (haydnba) [inline src/model/database/db_build.js:13]: Isaac can decide what to do with this in the future

File: src/test/controllers/signup.test.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+// require test database build script
+const runDbBuild = require('../../model/database/db_build');
+
+test('Test signup GET route', (t) => {
+  request(router)
+    .get('/signup')
+    .expect(200)
+    .expect('Content-Type', /html/)
+    .end((err, res) => {
+      t.ok(res);
```
Comment (i2xzy) [inline src/test/controllers/signup.test.js]: extra t.ok

File: src/views/signup.hbs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<section>
+  <h1>Welcome to Well-Bean</h1>
+  <article>
+    <h3>Signup Here</h3>
+    <form action="" method="POST">
```
Comment (i2xzy) [inline src/views/signup.hbs]: add a route

**Pull request merged**